src/backend/distributed/sql/udfs/citus_schemas/latest.sql

```diff
@@ -0,0 +1,33 @@
+DO $$
+declare
+citus_schemas_create_query text;
+BEGIN
+citus_schemas_create_query=$CSCQ$
+    CREATE OR REPLACE VIEW %I.citus_schemas AS
+    SELECT
+        schemaid::regnamespace AS schema_name,
+        colocationid AS colocation_id,
+        (
+            SELECT pg_size_pretty(sum(citus_total_relation_size(logicalrelid)))
+            FROM pg_dist_partition pdp WHERE pdp.colocationid = ts.colocationid
+        ) AS schema_size,
```

Comment:
should we say total_relation_size instead of schema_size because this doesn't account for sequences, indexes etc.?

Reply:
We discussed offline and decided to continue with schema_size